cmd/mysterium_node/mysterium_node.go

```diff
@@ -89,3 +95,52 @@ func NewCommand() (*cli.App, error) {
 
 	return app, nil
 }
+
+func newApp() (*cli.App, error) {
+	app := cli.NewApp()
+	return app, config.RegisterFlagsNode(&app.Flags)
+}
+
+// uiCommands is a map which consists of all
+// commands are used directly by a user.
+var uiCommands = map[string]struct{}{
+	command_cli.CliCommandName: {},
+}
+
+// readyOnceFunc returns a func which is only run once and can
+// be executed to preconfigure global settings used in the app.
+func readyOnceFunc() cli.BeforeFunc {
+	var once sync.Once
+	return func(ctx *cli.Context) error {
+		once.Do(func() {
+			cmd := ctx.Args().First()
+			if _, ok := uiCommands[cmd]; !ok {
+				// If the command is not meant for user
+				// interaction, skip.
+				return
+			}
+
+			logDir := ctx.String(config.FlagLogDir.Name)
+			if logDir == "" {
+				// Dont configure UI logger if we have
+				// no log dir.
+				return
+			}
+
+			level, err := zerolog.ParseLevel(ctx.String(config.FlagLogLevel.Name))
+			if err != nil {
+				level = zerolog.DebugLevel
+			}
+
+			err = logconfig.ConfigureUI(&logconfig.LogOptions{
```

Comment:
How about `logconfig.BootstrapCLI(...)` - package gives away the context and function name can explain the rest?